src/lib/booleans.js

```diff
@@ -0,0 +1,62 @@
+exports.negate = a => {
+  return !a;
+};
+
+exports.both = (a, b) => {
+  return a && b;
+};
+
+exports.either = (a, b) => {
+  return a || b;
+};
+
+exports.none = (a, b) => {
+  if (!a && !b) {
+    return true;
+  }
+  return false;
+};
+
+exports.one = (a, b) => {
+  return a != b;
+};
+
+exports.truthiness = a => {
+  return !!a;
+};
+
+exports.isEqual = (a, b) => {
+  return a == b;
```

Comment:
Should use triple equals for comparisons. Double equals will allow coercion. So for eg '1' == 1 is true but '1' === 1 is false